multi_import/multi_importer.py

```diff
@@ -85,9 +85,20 @@ def import_data(self, data):
 
         bound_importers = self._transform_multi_input(data)
 
-        for importer, filename, dataset in bound_importers:
-            result = importer.import_data(dataset, context, transaction=False)
-            results.add_result(filename, result)
+        for importer, datasets in bound_importers:
+            serializer_context = importer.get_serializer_context(context)
+
+            read_datasets = [
+                (filename, importer.read_rows(dataset))
+                for filename, dataset in datasets
+            ]
+
+            for filename, rows in read_datasets:
+                importer.load_instances(rows, serializer_context)
+
+            for filename, rows in read_datasets:
+                result = importer.process_rows(rows, serializer_context)
+                results.add_result(filename, result)
```

Comment:
Right OK, so just to test my understanding: if it WERE the case that there already existed an object with that pk, then the current code (i.e. before this mr) would have rightfully displayed an error.  Its just the case of NEW objects with NEW uuid's that the old code failed to handle, and this update fixes, yes?